ops/ansible/playbooks-ccs/rotate_bfd_postgres_user.yml

```diff
@@ -0,0 +1,26 @@
+#ansible-playbook rotate_bfd_postgres_user.yml --extra-vars "rds_database= master_user= master_pw= old_user= db_user= db_user_pw= rds_region="
```

Comment:
How is the old_user var utilized here?

Reply:
The old_user variable was supposed to dropped, I switched to db_user as I felt like it made more logical sense.